package.json

```diff
@@ -51,7 +51,7 @@
     "start": "npm run build && concurrently -n tsc,server \"tsc --watch --preserveWatchOutput --outDir 'build/src' --declarationDir 'build/declarations'\" \"http-server ./ -s -o /tests/playground.html -c-1\"",
     "tsc": "gulp tsc",
     "test": "gulp test",
-    "test:browser": "npx mocha ./tests/browser/test --config ./tests/browser/test/.mocharc.js",
+    "test:browser": "cd tests/browser && npx mocha",
```

Comment:
Consider adding `&& cd ..` to get back to the original directory after running the command.